ndcube/utils/__init__.py

```diff
@@ -1 +1,3 @@
 from . import collection, cube, misc, sequence, wcs
```

Comment:
Can I ask why this was done?

Reply:
I actually don't know...  I'm happy for it to be changed if it doesn't break any tests.

Reply:
It is so someone cna import ndcube.utils and get access to the entire range of modules without having to import them directly. This will stay.